### DISCUSSION TITLE
Question:How to sync a saelor project on git repo

### DISCUSSION THREAD

**Discussion by SpyrosMourelatos** [saleor/saleor#8623] [Deployment]
Hi there and thanks for the awesome project.
I just started a new saleor project with the instructions mentioned [here]( https://docs.saleor.io/docs/3.0/developer/installation)
Everything worked fine.
Then I uploaded it to a git repository and said to my colleague to just run `docker-compose build` and `docker-compose up` but as you can imagine it didn't work as he didn't had my database on his computer. I would like to ask how to work collaboratively with saleor on git and how to migrate the database.
Thank you in advance

**Comment by karolkielecki** [comment]
@SpyrosMourelatos Hi! Thanks for your kind words. You can find a solution under this discussion: https://github.com/saleor/saleor/discussions/8487. Good luck!
